p2p/node/pubsubManager/gossipsub.go

```diff
@@ -51,7 +51,8 @@ func NewGossipSubManager(ctx context.Context, h host.Host) (*PubsubManager, erro
 	cfg.Dlo = 6
 	cfg.Dhi = 45
 	cfg.Dout = 20
-	ps, err := pubsub.NewGossipSub(ctx, h, pubsub.WithGossipSubParams(cfg))
+	MaxSize := 3 << 20
```

Comment:
Recommend to make this a protocol param

Reply:
Its the only one in this list that would cause a hardfork.

Reply:
Moved it to protocol params for easy access